_overviews/cheatsheets/index.md

```diff
@@ -26,270 +26,271 @@ permalink: /cheatsheets/index.html
       <br><em><strong>same as</strong></em><br>
       <pre class="highlight"><code>if (check) happy else ()</code></pre></td>
-      <td>conditional sugar.</td>
+      <td>Conditional sugar.</td>
     </tr>
     <tr>
-      <td><pre class="highlight"><code>while (x &lt; 5) { 
+      <td><pre class="highlight"><code>while (x &lt; 5) {
   println(x)
   x += 1
 }</code></pre></td>
-      <td>while loop.</td>
+      <td>While loop.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>do {
   println(x)
   x += 1
 } while (x &lt; 5)</code></pre></td>
-      <td>do while loop.</td>
+      <td>Do-while loop.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>import scala.util.control.Breaks._
+
 breakable {
-  for (x <- xs) {
-    if (Math.random < 0.1)
+  for (x &lt;- xs) {
+    if (Math.random &lt; 0.1)
       break
   }
 }</code></pre></td>
-      <td>break. (<a href="http://www.slideshare.net/Odersky/fosdem-2009-1013261/21">slides</a>)</td>
+      <td>Break (<a href="http://www.slideshare.net/Odersky/fosdem-2009-1013261/21">slides</a>).</td>
     </tr>
     <tr>
-      <td><pre class="highlight"><code>for (x &lt;- xs if x%2 == 0)
-yield x * 10</code></pre>
+      <td><pre class="highlight"><code>for (x &lt;- xs if x % 2 == 0)
+  yield x * 10</code></pre>
       <br><em><strong>same as</strong></em><br>
-      <pre class="highlight"><code>xs.filter(_%2 == 0).map( _ * 10)</code></pre></td>
-      <td>for comprehension: filter/map</td>
+      <pre class="highlight"><code>xs.filter(_ % 2 == 0).map(_ * 10)</code></pre></td>
+      <td>For comprehension: filter/map.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>for ((x, y) &lt;- xs zip ys)
-yield x * y</code></pre>
+  yield x * y</code></pre>
       <br><em><strong>same as</strong></em><br>
       <pre class="highlight"><code>(xs zip ys) map {
   case (x, y) =&gt; x * y
 }</code></pre></td>
-      <td>for comprehension: destructuring bind</td>
+      <td>For-comprehension: destructuring bind.</td>
     </tr>
     <tr>
       <td><pre class="highlight"><code>for (x &lt;- xs; y &lt;- ys)
-yield x * y</code></pre>
+  yield x * y</code></pre>
       <br><em><strong>same as</strong></em><br>
       <pre class="highlight"><code>xs flatMap { x =&gt;
   ys map { y =&gt;
     x * y
   }
 }</code></pre></td>
-      <td>for comprehension: cross product</td>
+      <td>For-comprehension: cross product.</td>
     </tr>
     <tr>
-      <td><pre class="highlight"><code>for (x <- xs; y <- ys) {
+      <td><pre class="highlight"><code>for (x &lt;- xs; y &lt;- ys) {
   val div = x / y.toFloat
   println("%d/%d = %.1f".format(x, y, div))
 }</code></pre></td>
-      <td>for comprehension: imperative-ish<br /><a href="http://java.sun.com/javase/6/docs/api/java/util/Formatter.html#syntax">sprintf-style</a></td>
+      <td>For-comprehension: imperative-ish.<br /><a href="http://java.sun.com/javase/6/docs/api/java/util/Formatter.html#syntax">Sprintf-style</a>.</td>
```

Comment:
This looks strange.  Maybe keep `sprintf` as lowercase, since it's a function name?

Reply:
Then with `<code>`?